figures/metrics.py

```diff
@@ -26,11 +26,14 @@
 import math
 
 from django.contrib.auth import get_user_model
-from django.db.models import Avg, Max
+from django.db.models import Avg, Max, Q
 
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from courseware.courses import get_course_by_id  # pylint: disable=import-error
 from courseware.models import StudentModule  # pylint: disable=import-error
 
+import organizations
```

Comment:
Is it figures version of organizations or any package? If it's figures version of organization kindly name it figures_organizations to make it more verbose.

Reply:
@iam-mhaseeb for MVP we are going with minimum changes in Figures APP.

Reply:
I mean use it like `import organizations as figures_organizations` to make it more verbose.

Reply:
@iam-mhaseeb this is edx organization. Figures don't have its organization model.

Reply:
Then edx_organizations would be good fit here. It looks like organizations belong to figures. @zubair-arbi your thoughts here?

Reply:
If we rename the `organization` as `edx_organizations` then we have update it in all views using the `organization` thus creating additional changes in figures while our approach is to go with minimum changes,